design/nfs/nfs-quota.md

```diff
@@ -0,0 +1,68 @@
+# NFS Quota
+
+## Background
+
+Currently, when the user creates PersistentVolumes from an NFS Rook share/export via PersistentVolumeClaim, the provisioner does not provide the spesific capacity as requested. For example the users create PersistentVolumeClaim as following:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: rook-nfs-pv-claim
+spec:
+  storageClassName: "rook-nfs-share"
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Mi
+```
+
+The client still can use the higher capacity than `1mi` as requested.
+
+This proposal is to add features which the Rook NFS Provisioner will provide the specific capacity as requested from `.spec.resources.requests.storage` field in PersistentVolumeClaim.
+
+## Implementation
+
+The implementation will be use `Project Quota` on xfs filesystem. When the users need to use the quota feature they should use xfs filesystem with `prjquota/pquota` mount options for underlying volume. Users can specify filesystem type and mount options through StorageClass that will be used for underlying volume. For example:
+
+```yaml
```

Comment:
ah, this is the *underlying* volume that is underneath the NFS share/export, NOT the storageClass that is used for a NFS based PVC.  I was confused at first, even though you did mention "underlying volume" in the sentence before. 

this may be helped from more clarity (or a little diagram), of the layering of volumes that are in play here
* NFS PV
* NFS share/export
* underlying volume